kahuna/public/js/preview/image.html

```diff
@@ -121,10 +121,8 @@
                 <gr-icon>today</gr-icon>
             </div>
 
-            <div class="bottom-bar__action"
-                 ng-if="ctrl.states.isSyndicated"
-                 title="Image is available to syndication partners">
-                <gr-icon>monetization_on</gr-icon>
+            <div class="bottom-bar__action" ng-if="ctrl.states.syndicationStatus !== 'unsuitable'">
```

Comment:
`syndicationStatus === 'unsuitable'` when there is no RCS information for the image (anything that has not been used).

Don't show the icon in this case, to keep the UI from getting too busy as there are more unused images than used!